ingress/controllers/nginx/examples/default/rc-default.yaml

```diff
@@ -14,11 +14,18 @@ spec:
         k8s-app: nginx-ingress-lb
         name: nginx-ingress-lb
     spec:
-      terminationGracePeriodSeconds: 60      
+      terminationGracePeriodSeconds: 60
       containers:
       - image: gcr.io/google_containers/nginx-ingress-controller:0.8.2
         name: nginx-ingress-lb
         imagePullPolicy: Always
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 80
```

Comment:
The nginx controller also serves a /healthz: https://github.com/kubernetes/contrib/blob/master/ingress/controllers/nginx/nginx.tmpl#L289
I want the readiness to be on :80, because that's the port that matches the nodeport on the service, so the GCE ingress controller will adopt that as a cloud health check (for this to happen the nodePort's target port and the readiness probe must be the same port).